engine/src/main/java/org/terasology/rendering/world/WorldRenderer.java

```diff
@@ -33,7 +32,7 @@
     void onChunkUnloaded(Vector3i chunkPos);
 
     public enum WorldRenderingStage {
-        DEFAULT,
+        MONO,
         OCULUS_LEFT_EYE,
         OCULUS_RIGHT_EYE
```

Comment:
Changed to LEFT_EYE and RIGHT_EYE as you recommended @msteiger. Of course this stills leaves out research systems designed to present a Virtual Reality to lab spiders, who are notoriously much more multi-eyed. We might have to create an issue about that.